### DISCUSSION TITLE
Obtain a Luminance FITS image from ARW

### DISCUSSION THREAD

**Discussion by RhoLuke** [ImageMagick/ImageMagick#5600] [Consulting]
I have to extract the Luminance of the ARW image and convert it to a FITS format. With "Luminance" I mean the sum of the channels RGB. Possibly in the fastest possible way.

Thanks

**Comment by snibgo** [comment]
What version of IM, on what platform?

There are many ways of doing this, such as:
```
magick in.arw -channel RGB -separate +channel -evaluate-sequence Add out.fits
```
... or ...
```
magick in.arw -fx r+g+b out.fits
```
I don't know which method would be the fastest.

If your IM is older than 7.1.0-21, the "-fx" method will be slow.

**Comment by RhoLuke** [comment]
HI,
I'm using IM (v.7.1.0-50) on Raspberry Pi4 (4Gb ram).

Thanks,

 Luca.
